.github/workflows/continuous_integration.yml

```diff
@@ -50,11 +50,32 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
 
-  multiarch_build:
+  snapshot_release:
```

Comment:
nit: I know that the release workflows will fail here regardless but we can probably just remove them in our fork right?